src/java/org/apache/cassandra/metrics/StorageMetrics.java

```diff
@@ -32,4 +34,13 @@ public class StorageMetrics
     public static final Counter exceptions = Metrics.counter(factory.createMetricName("Exceptions"));
     public static final Counter totalHintsInProgress  = Metrics.counter(factory.createMetricName("TotalHintsInProgress"));
     public static final Counter totalHints = Metrics.counter(factory.createMetricName("TotalHints"));
+
+    /* Tracks the most filled data_file_directory */
+    public static final Gauge<Double> diskUsed = Metrics.register(factory.createMetricName("DiskUsed"), new Gauge<Double>()
```

Comment:
This variable is not used anywhere, correct? If so, it might be cleaner to just register the metric inside a static block:
```
static {
     Metrics.register(...)
}
```

Reply:
Really hate the idea of putting this in a static block because its not used. There are plenty of places where it will improve logging insight so I'll add it there.